cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -32,6 +32,8 @@
 
     boolean usePreemptibleVms();
 
+    boolean useScratchSsds();
```

Comment:
out of curiosity, what does "scratch" mean exactly? we've been referring to them as local ssds (as do the options in the gui). Does scratch somehow describe what we're doing more accurately?

Reply:
It's actually one of the two terms Google uses, I think it might be to make it clear they're transient, like a temp directory or working directory. Surprisingly Wikipedia has an article on it:

https://en.wikipedia.org/wiki/Scratch_space